apps/react-ts-remote/webpack.config.js

```diff
@@ -12,6 +12,10 @@ module.exports = composePlugins(
   withNx(),
   withReact(),
   async (config, context) => {
+    if (!config.devServer) {
+      config.devServer = {};
+    }
+    config.devServer.host = '127.0.0.1';
```

Comment:
Is `devServer` mutated or is `config` a deep-cloned version of the webpack/rspack config?